pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```
@@ -116,24 +124,54 @@ spec:
     description: The buildah version used to build the model container image
     value: $(tasks.retrieve-build-image-info.results.buildah-version)
   tasks:
+  - name: fetch-model-params
+    params:
+    - name: model-name
+      value: $(params.model-name)
+    - name: model-version
+      value: $(params.model-version)
+    - name: model-registry-hostname
+      value: $(params.model-registry-hostname)
+    - name: model-registry-enabled
+      value: $(params.model-registry-enabled)
+    - name: git-model-repo
+      value: $(params.git-model-repo)
+    - name: git-model-revision
+      value: $(params.git-model-revision)
+    - name: model-dir
+      value: $(params.model-dir)
+    - name: fetch-model
+      value: $(params.fetch-model)
+    - name: modelRelativePath
```

Just repeating, so it doesn't get lost. This is still not fixed:

> Missing:
> ```suggestion
>       value: $(params.modelRelativePath)
>     - name: s3-bucket-name
>       value: $(params.s3-bucket-name)
> ```
> 
> 
> 
> Also, it would be good to have the parameters in the same order both in the Pipeline here and in the Task.
> Another improvement can be to have the `s3-bucket-name` parameter without the default value in the task. This way, the pipeline would most likely fail that the pipeline is not passing the parameter to the task. But once properly linked, it won't be failing because the pipeline parameter `s3-bucket-name` itself has a default value of `""`.



Therefore, I would:
1. Make the order of params here and in the task definition same.
2. Remove any default values for the task params, so if any of the param is not set by the pipeline, it will fail.
3. Add the `s3-bucket-name` parameter here as it is still missing.

---

Done all three steps. Thanks for catching this @MarianMacik